api/v1alpha2/subnamespace_anchor.go

```diff
@@ -68,9 +68,22 @@ type SubnamespaceAnchor struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 
+	Spec   SubnamespaceAnchorSpec   `json:"spec,omitempty"`
 	Status SubnamespaceAnchorStatus `json:"status,omitempty"`
 }
 
+type SubnamespaceAnchorSpec struct {
+	// Lables is a list of labels and values to apply to the current namespace and all of its
+	// descendants. All label keys must match a regex specified on the command line by
+	// --managed-namespace-label.
```

Comment:
"All label keys must be managed labels (see HNC docs) and must match a regex..."

Similar comments for annotations